packages/data-explorer-ui/src/components/Export/components/ExportEntityToTerra/exportEntityToTerra.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ElementType } from "react";
```

Comment:
ExportEntityToTerra component takes a number of props all defined in the `Data-Browser` such as hooks and the form - to allow for complete flexibility and re-use.